src/some_socket_addr.rs

```diff
@@ -27,6 +30,8 @@ impl Display for SomeSocketAddr {
             SomeSocketAddr::Unix(_x) => "unix".fmt(f),
             #[cfg(feature = "inetd")]
             SomeSocketAddr::Stdio => "stdio".fmt(f),
+            #[cfg(all(any(target_os = "linux", target_os = "android", target_os = "macos"), feature = "vsock"))]
+            SomeSocketAddr::Vsock(x) => x.fmt(f),
```

Comment:
It would produce something like `cid: {} port: {}` (without any direct indication that it is VSOCK). Is it OK?

Reply:
Done as  "vsock:{cid}:{port}"